internal/cmd/object-storage/credentials/create/create.go

```diff
@@ -0,0 +1,128 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	objectStorageUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/client"
+	"github.com/stackitcloud/stackit-sdk-go/services/objectstorage"
+)
+
+const (
+	expiresFlag          = "expires"
+	credentialsGroupFlag = "credentials-group"
```

Comment:
The flag should be `credentials-group-id` to be more clear

Then the variable name should also be updated to `credentialsGroupIdFlag`